apps/site/assets/css/_schedule-page.scss

```diff
@@ -363,13 +376,29 @@
   font-variant-numeric: tabular-nums;
 }
 
-.schedule-finder__first-last-trip {
+.schedule-finder__first-last-trip--bordered {
   border-top: 3px solid $gray-lightest;
+  padding: ($base-spacing / 2) 0;
+}
+
+.schedule-finder__first-last-trip-label {
+  @extend .u-small-caps; // scss-lint:disable PlaceholderInExtend
+  @extend .u-bold; // scss-lint:disable PlaceholderInExtend
```

Comment:
On the chance that we decide in the future to change our bold style (say by tweaking the font-weight from bold to bolder, or adjusting font-size, or any other change) by redefining `.u-bold`, extending the class here will help us stay consistent.

Reply:
OK — is there a more-recommended way for us to do this, such that we wouldn't have to suppress a warning?

Reply:
The config is [expecting](https://github.com/sds/scss-lint/blob/master/lib/scss_lint/linter/README.md#placeholderinextend) us to extend a [placeholder](https://sass-lang.com/documentation/style-rules/placeholder-selectors) and use that instead of extending the class. 

We have a handful of placeholder selectors scattered across our code base, though most of them are only used in the file they are defined in.

Some options:
1) Use these two extended classes in the HTML markup (likely the intended usage of such a class and easy enough to do for this selector).
2) Update the scss-lint config to disable that particular check (easy, and we can remove 3 of ~30 instances of `// scss-lint:disable`).
3) Write and use placeholders instead (this will work for this case, but still leaves us high and dry in the case where we want to extend an existing class from, say, Bootstrap -- which admittedly doesn't happen very often).

Reply:
It seems like there are [legitimate pitfalls](http://8gramgorilla.com/mastering-sass-extends-and-placeholders/) to extending selectors, so I don't think continuing to disable the linter check is the best option in the long run. I like the idea of using placeholders since that's the recommended alternative, but I'm not sure how much work it would be to replace all our existing instances of `@extend .selector`. If it's non-trivial, maybe we can file that as a separate tech debt task?